scripts/parse_existing_cohort.py

```diff
@@ -210,6 +212,12 @@ def get_existing_external_sequence_ids(self, participant_map: dict[str, dict]):
     '--project',
     help='The metamist project to import manifest into',
 )
+@click.option(
+    '--sequencing-type',
+    type=click.Choice(['genome', 'exome']),
+    required=True,
```

Comment:
Do we need to set required to be true? Esp since we have a default 'genome' described below

Reply:
Good call!